### PR TITLE
Let seed template automatically find the latest version

### PR DESCRIPTION
**Pull request by octonato**

### CODE REVIEW

File: src/main/g8/default.properties [hunk 1]
```diff
@@ -2,3 +2,4 @@ description = This template generates a Play Scala project
 name=play-scala-seed
 organization=com.example
 verbatim=*.css *.js *.png logback.xml
+play_version = maven(com.typesafe.play, sbt-plugin)
```
Comment (octonato) [inline src/main/g8/default.properties]: We don't need to keep updating the version, we can let is be automatically fetched from maven central.

Comment (mkurz) [reply src/main/g8/default.properties]: Nice!

**Review by mkurz — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -6,8 +6,8 @@ env:
   - TRAVIS_JDK=11
 
 before_install: curl -Ls https://git.io/jabba | bash && . ~/.jabba/jabba.sh
-install: jabba install "adopt@~1.$TRAVIS_JDK.0-0" && jabba use "$_" && java -Xmx32m -version
-script: scripts/test-sbt
+#install: jabba install "adopt@~1.$TRAVIS_JDK.0-0" && jabba use "$_" && java -Xmx32m -version
+#script: scripts/test-sbt
```
Comment (octonato) [inline .travis.yml]: @mkurz, the tests are broken. I'm disabling it for now.

Comment (mkurz) [reply .travis.yml]: :+1:

Comment (octonato) [reply .travis.yml]: hmm, I will revert it. Seems to be something on my machine.

File: project/plugins.sbt [hunk 3]
```diff
@@ -1 +1 @@
-addSbtPlugin("org.foundweekends.giter8" % "sbt-giter8" % "0.11.0")
+addSbtPlugin("org.foundweekends.giter8" % "sbt-giter8" % "0.13.1")
```
Comment (octonato) [inline project/plugins.sbt:1]: we need to update the gitter8 version in order to use `maven(com.typesafe.play, sbt-plugin, stable)` in the test

**Pull request merged**